src/Orleans/Messaging/IGatewayListProvider.cs

```diff
@@ -15,8 +15,7 @@ public interface IGatewayListProvider
         /// Initializes the provider, will be called before all other methods
         /// </summary>
         /// <param name="clientConfiguration">the given client configuration</param>
-        /// <param name="logger">the logger to be used by the provider</param>
-        Task InitializeGatewayListProvider(ClientConfiguration clientConfiguration, Logger logger);
+        Task InitializeGatewayListProvider(ClientConfiguration clientConfiguration);
```

Comment:
removed Logger from IGatewayListProvider.Init. now it gets logger from DI in constructor injection